plugins/sigma/tests/tests.cpp

```diff
@@ -155,26 +163,62 @@ TEST(modifier - re) {
   CHECK_EQUAL(search_id, expected);
 }
 
+TEST(modifier - re 2) {
+  auto yaml = R"__(
+    foo|re: '.*foobar.*'
+  )__";
+  auto search_id = to_search_id(yaml);
+  auto expected = to_expr("foo == /.*foobar.*/");
+  CHECK_EQUAL(search_id, expected);
+}
+
+TEST(modifier - re - no wildcards) {
+  auto yaml = R"__(
+    foo|re: 'foobar'
+  )__";
+  auto search_id = to_search_id(yaml);
+  auto expected = to_expr("foo == /foobar/");
+  CHECK_EQUAL(search_id, expected);
+}
+
 TEST(modifier - startswith) {
   auto yaml = R"__(
     foo|startswith: "x"
   )__";
   auto search_id = to_search_id(yaml);
-  // TODO: blocked by VAST has pattern matching capability
-  // auto expected = to_expr("foo == /^x/");
-  auto expected = to_expr("foo ni \"x\"");
+  auto expected = to_expr("foo == /^x.*/i");
   CHECK_EQUAL(search_id, expected);
 }
 
+TEST(modifier - startswith - proper backslash escaping) {
+  auto yaml = R"__(
+    foo|startswith: "C:\rundll32"
+  )__";
+  auto search_id = to_search_id(yaml);
+  auto expected = to_expr(R"(foo == /^C:\rundll32.*/i)");
+  CHECK_EQUAL(search_id, expected);
+  auto str = to_string(expected);
+  CHECK_EQUAL(str, "foo == /^C:\\rundll32.*/i");
```

Comment:
Can you explain the escaping difference? I'm not understanding why only the second `rundll` example has a double `\\`.